build-system/pr-check.js

```diff
@@ -416,7 +419,9 @@ function runAllCommands() {
   if (process.env.BUILD_SHARD == 'integration_tests') {
     command.cleanBuild();
     command.buildRuntimeMinified(/* extensions */ true);
-    command.runBundleSizeCheck();
+    if (checkBundleSize) {
+      command.runBundleSizeCheck();
+    }
```

Comment:
How about eliminating the `checkBundleSize` param, and simply doing this inside `runAllCommands`?
```
    // Disable bundle-size check on release branch builds.
    if (process.env.TRAVIS_BRANCH === 'master') {
      command.runBundleSizeCheck();
    }
```

Reply:
Done.